sim/common/cata/enchant_effects.go

```diff
@@ -380,6 +380,63 @@ func init() {
 		character.ItemSwap.RegisterOnSwapItemForEnchantEffect(4115, aura)
 	})
 
+	// Enchant: 4116, Spell: 75175 - Darkglow Embroidery
+	core.NewEnchantEffect(4116, func(agent core.Agent) {
+		character := agent.GetCharacter()
+
+		statAura := character.NewTemporaryStatsAura(
+			"Darkglow Embroidery Cata",
+			core.ActionID{SpellID: 75175},
```

Comment:
Can you use https://www.wowhead.com/cata/spell=75173/darkglow for the statAura instead so it matches the actual buff you get?

Reply:
pushed changes